packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -86,15 +88,16 @@ export class TSClient implements Generatable {
       engineVersion: this.options.engineVersion,
       datasourceNames: datasources.map((d) => d.name),
       activeProvider: this.options.activeProvider,
+      dataProxy: this.options.dataProxy,
```

Comment:
`dataProxy` is now stored in the generated config instead of being in the generator config block (via schema).